src/descriptor/policy.rs

```diff
@@ -811,6 +817,12 @@ impl<Ctx: ScriptContext> ExtractPolicy for Miniscript<DescriptorPublicKey, Ctx>
                         csv: None,
                     },
                 };
+                if let BuildSatisfaction::PsbtTimelocks { current_height, .. } = build_sat {
+                    let after = After::new(Some(current_height), false);
+                    if Satisfier::<bitcoin::PublicKey>::check_after(&after, *value) {
+                        policy.satisfaction = policy.contribution.clone();
```

Comment:
Same as the older, you should also run the `check_after()` from `PsbtInputSatisfier`

Reply:
should be fixed in 6558c74d485df0d0bcd64a1ca108c40b5e5a04da